src/adapter.ts

```diff
@@ -1,16 +1,103 @@
 import { MoveCallTransaction, SuiTransactionResponse } from '@mysten/sui.js';
 import { ConnectResponseType, MartianApis, Permission, SignMessageResponseType } from './types';
+import { Wallet, SUI_DEVNET_CHAIN, SUI_TESTNET_CHAIN, IdentifierArray, WalletAccount, ConnectOutput, SuiSignAndExecuteTransactionInput, SuiSignAndExecuteTransactionOutput } from "@mysten/wallet-standard";
+import { WalletVersion, WalletIcon } from '@wallet-standard/base';
+import { ReadonlyWalletAccount } from "@mysten/wallet-standard";
+// import { WalletAdapter } from "@mysten/wallet-adapter-base";
+
+import {
+  ConnectFeature,
+  ConnectMethod,
+  EventsFeature,
+  EventsOnMethod,
+  SuiSignAndExecuteTransactionFeature,
+  SuiSignAndExecuteTransactionMethod
+} from "@mysten/wallet-standard";
 
 declare const window: {
```

Comment:
please remove this, this is replacing actual martian object

Reply:
Deleting this gives error in lines 156, 160

Reply:
Ah, i think we can declare type instead? Like type window {sui: any} something like this